druid-shell/examples/invalidate.rs

```diff
@@ -78,6 +78,10 @@ impl WinHandler for InvalidateTest {
         }
     }
 
+    fn destroy(&mut self) {
+        Application::global().quit()
+    }
+
```

Comment:
This looks unrelated - I assume this is for some future further additions to this example?

Reply:
Well this `invalidate` example never quits when the window is closed. The other druid-shell examples quit on window closing. This simple change fixes that.